custom_components/tuya_local/devices/aygrochy_inline_wifi_air_vent.yaml

```diff
@@ -0,0 +1,314 @@
+name: Aygrochy fan
+primary_entity:
+  entity: climate
+  dps:
+    - id: 2
+      name: preset_mode
```

Comment:
My suggestion is to duplicate this to hvac_mode, and map "ON" to fan_only, "OFF" to "off", "TIMER" to "auto" but marked as hidden, "AUTO" to "auto", and "ALARM" to "fan_only" but marked as hidden.
The target temperatures should have conditional mappings based on this mode to select the appropriate dps for the mode (using `value_redirect`).

Reply:
I did say duplicate.

You will still need it accessible as preset_mode to switch between the modes that have been merged to fit into the limited options of hvac_mode.

Reply:
sorry misinterpreted, duplicated now